src/drivers/actuators/voxl_esc/voxl_esc.cpp

```diff
@@ -1240,11 +1254,56 @@ bool VoxlEsc::updateOutputs(bool stop_motors, uint16_t outputs[MAX_ACTUATORS],
 			sizeof(cmd.buf),
 			_extended_rpm);
 
+
 	if (_uart_port->uart_write(cmd.buf, cmd.len) != cmd.len) {
 		PX4_ERR("VOXL_ESC: Failed to send packet");
 		return false;
 	}
 
+	// Track and manage gpio command writes
+    	bool write_gpio_command = false;
```

Comment:
In the source file this line has some spaces and a tab before it and the alignment is off

Reply:
fixed